### PR TITLE
Add linters to ci

### PR DESCRIPTION
**Pull request by deepbrook**
Hey there,

as threatened via DevRant, here's my PR ;)

I've added linters to the travis yml and added linter fixes to the code. I have also removed imports from builtins, because this is not necessary to my knowledge.

Additionally, you should consider dropping support for python 2 - it's deprecated (and dies 2019) and its use should not be encouraged anymore. It also makes the code much more maintainable. 

Let me know what you think about this.

### CODE REVIEW

**Comment by deepbrook** [comment]
Ah yes, running tests should've been done as well. I'm on that, linting and its fixes for unittests
 included ;)

File: pirant/urlbuilder.py [hunk 1]
```diff
@@ -48,6 +48,7 @@ def get_collabs_url(self, skip, limit):
         skip = self.validate_int_input(skip)
         return str(self.collabs_url % (self.base_url, self.app_version, skip, limit))
 
+    @staticmethod
```
Comment (deepbrook) [inline pirant/urlbuilder.py]: This right here seems odd. 
Python, by default, pases `self` to class methods that are being called on an instance. I've removed this again in commit 62f4f2372766cafb2c6c5265ae0ff5ad4dbda8ca - the tests passed, but check for yourself if this is ok to do - perhaps I oversaw something that fall between the cracks of the test suite.

**Comment by aayush26** [comment]
Thanks for the contribution as mentioned in devrant. I am reviewing it and will get back to you.

**Comment by aayush26** [comment]
LGTM

**Pull request merged**